src/aspire/utils/numeric/scipy_fft.py

```diff
@@ -1,14 +1,14 @@
-from scipy import fftpack
+import scipy.fft as spfft
```

Comment:
Same here. `from scipy import fft` is enough.

Reply:
The module name `fft` is in conflict with the function `fft` name. Changed to use  `import scipy as sp` and `sp.fft`.